tracing/src/main/java/com/palantir/tracing/DetachedSpan.java

```diff
@@ -69,7 +70,33 @@ static DetachedSpan start(
      * instead of thread state.
      */
     @MustBeClosed
-    CloseableSpan childSpan(String operationName, SpanType type);
+    default CloseableSpan childSpan(String operationName, SpanType type) {
```

Comment:
Call me old-fashioned, but these default methods scare me just a little bit. And the amount of them

Reply:
Ya, it's a balancing act.